.github/workflows/ci.yml

```diff
@@ -233,7 +233,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04, macos-12, windows-2019, windows-2022]
-        go-version: ["1.17.13", "1.18.10"]
+        go-version: ["1.17.13", "1.19.6"]
```

Comment:
do we still need 1.17😅?

Reply:
idk, some distro might be still building containerd with Go 1.17?

Reply:
Yeah, we shouldn't put too much effort into it, but I guess it's ok to have "oldest version (version at time of first release) and current version" to see "if it still builds" is okay to have.

Once it becomes really problematic, we can bump the oldest version to something newer.